FRW/Modules/Browser/View/BrowserAuthzView.swift

```diff
@@ -85,7 +85,12 @@ struct BrowserAuthzView: View {
 
             Spacer()
 
-            actionView
+            VStack(spacing: 0) {
+                InsufficientStorageToastView<BrowserAuthzViewModel>()
```

Comment:
Should we put the into the actionView instead of adding it in different places ?

Reply:
Applied similar refactoring to other places (`NFTTransferView` and `WalletSendAmountView`)